src/thunderbots/software/ai/navigator/trespass.cpp

```diff
@@ -1,8 +1,59 @@
 #include "ai/navigator/trespass.h"
 
+#include <exception>
+
+#include "geom/circle.h"
+#include "geom/point.h"
+#include "util/math_functions.h"
+
 namespace Navigator
 {
     namespace Trespass
     {
```

Comment:
You can save some indenting here by doing something like:
```
namespace Navigator::Trespass {
```

(it's a feature as of C++17)